chain-impl-mockchain/tests/common/address.rs

```diff
@@ -1,14 +1,30 @@
 use chain_addr::{Address, Discrimination, Kind, KindType};
-use chain_impl_mockchain::account::SpendingCounter;
-use chain_impl_mockchain::key::{EitherEd25519SecretKey, SpendingPublicKey};
+use chain_impl_mockchain::{
+    account::SpendingCounter,
+    key::{EitherEd25519SecretKey, SpendingPublicKey},
+    transaction::{Input, Output, UtxoPointer},
+    value::Value,
+};
+use std::fmt::{Debug, Formatter, Result};
 
+#[derive(Clone)]
 pub struct AddressData {
```

Comment:
Can you document the purpose this type is serving for tests, which could not be addressed by the code under test?